CONTRIBUTING.rst

```diff
@@ -59,60 +58,62 @@ Get Started!
 
 Ready to contribute? Here's how to set up `cubids` for local development.
 
-1. Fork the `cubids` repo on GitHub.
-2. Clone your fork locally::
+1.  Fork the `cubids` repo on GitHub.
+2.  Clone your fork locally::
 
     $ git clone git@github.com:your_name_here/cubids.git
 
-3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+3.  Install your local copy into a virtualenv.
+    Assuming you have virtualenvwrapper installed,
+    this is how you set up your fork for local development::
 
     $ mkvirtualenv cubids
     $ cd cubids/
     $ python setup.py develop
 
-4. Create a branch for local development::
+4.  Create a branch for local development::
 
     $ git checkout -b name-of-your-bugfix-or-feature
 
    Now you can make your changes locally.
 
-5. When you're done making changes, check that your changes pass flake8 and the
-   tests, including testing other Python versions with tox::
+5.  When you're done making changes, check that your changes pass flake8 and the
+    tests, including testing other Python versions with tox::
 
     $ flake8 cubids tests
     $ python setup.py test or pytest
     $ tox
 
-   To get flake8 and tox, just pip install them into your virtualenv.
+    To get flake8 and tox, just pip install them into your virtualenv.
 
-6. Commit your changes and push your branch to GitHub::
+6.  Commit your changes and push your branch to GitHub::
 
     $ git add .
     $ git commit -m "Your detailed description of your changes."
     $ git push origin name-of-your-bugfix-or-feature
 
-7. Submit a pull request through the GitHub website.
+7.  Submit a pull request through the GitHub website.
 
 Pull Request Guidelines
 -----------------------
 
 Before you submit a pull request, check that it meets these guidelines:
 
-1. The pull request should include tests.
-2. If the pull request adds functionality, the docs should be updated. Put
-   your new functionality into a function with a docstring, and add the
-   feature to the list in README.rst.
-3. The pull request should work for Python 3.5, 3.6, 3.7 and 3.8, and for PyPy. Check
-   https://circleci.com/gh/PennLINC/CuBIDS
-   and make sure that the tests pass for all supported Python versions.
+1.  The pull request should include tests.
+2.  If the pull request adds functionality, the docs should be updated. Put
+    your new functionality into a function with a docstring, and add the
+    feature to the list in README.rst.
+3.  The pull request should work for Python 3.5, 3.6, 3.7 and 3.8, and for PyPy.
```

Comment:
Do we want to keep supporting 3.5? Other than pybids the only important dependency is sklearn and I'm not sure how far back they go